modules/appeals_api/app/models/appeals_api/supplemental_claim.rb

```diff
@@ -31,6 +31,114 @@ def self.date_from_string(string)
       if: proc { |a| a.form_data.present? }
     )
 
+    def veteran_first_name
+      auth_headers.dig('X-VA-First-Name')
+    end
+
+    def veteran_middle_initial
+      auth_headers.dig('X-VA-Middle-Initial')
+    end
+
+    def veteran_last_name
+      auth_headers.dig('X-VA-Last-Name')
+    end
+
+    def full_name
+      "#{veteran_first_name} #{veteran_middle_initial} #{veteran_last_name}".squeeze(' ').strip
+    end
+
+    def ssn
+      auth_headers.dig('X-VA-SSN')
+    end
+
+    def file_number
+      auth_headers.dig('X-VA-File-Number')
+    end
+
+    def veteran_dob_month
+      birth_date.strftime '%m'
+    end
+
+    def veteran_dob_day
+      birth_date.strftime '%d'
+    end
+
+    def veteran_dob_year
+      birth_date.strftime '%Y'
+    end
+
+    def veteran_service_number
+      auth_headers.dig('X-VA-Service-Number')
+    end
+
+    def insurance_policy_number
+      auth_headers.dig('X-VA-Insurance-Policy-Number')
+    end
+
+    def mailing_address_number_and_street
+      veteran.dig('address', 'addressLine1') || ''
+    end
+
+    def mailing_address_apartment_or_unit_number
+      veteran.dig('address', 'addressLine2') || ''
+    end
+
+    def mailing_address_box
+      veteran.dig('address', 'addressLine3') || ''
+    end
+
+    def mailing_address_city
+      veteran.dig('address', 'city') || ''
+    end
+
+    def mailing_address_city_and_box
```

Comment:
Shouldn't the addressline3 be a part of the address and not the city?  Doesn't seem correct to have `Cincinnati PO Box 9` in the city field.